ci/testkomodo.sh

```diff
@@ -72,11 +72,12 @@ start_tests () {
     fi
     if ! which bsub 2>/dev/null && basetemp=$(mktemp -d -p ~/pytest-tmp); then
         export PATH=$PATH:/opt/pbs/bin
-        if [[ $(uname -r) == *el7* ]] ; then
-            export _ERT_TESTS_DEFAULT_QUEUE_NAME=permanent
-        else
-            export _ERT_TESTS_DEFAULT_QUEUE_NAME=permanent_8
-        fi
+        echo '#!/bin/sh' > queue_tester.sh
+        echo echo I am up >> queue_tester.sh
+        chmod 755 queue_tester.sh
+        for candidate in "permanent permanent_8 hb120 hb120_8 short normal"; do
```

Comment:
The old script checked for rhel8. Do we have still have to account for that?

Reply:
Yes we do. I had a hope tha this test would implicitly autodetect rhel7 vs rhel8 but I was not successful.